Libplanet/Blockchain/BlockChain.cs

```diff
@@ -600,32 +591,34 @@ internal void Swap(BlockChain<T> other)
                 {
                     return null;
                 }
-            }).ToArray();
-            IAccountStateDelta lastStates = tuples.Any()
-                ? tuples[tuples.Length - 1].Item2.OutputStates
-                : null;
+            }).Select(t => t.Item2).ToArray();
+        }
 
+        private void SetStateFromEvaluations(
```

Comment:
We'd better use the plural term “states” here for consistency (with `SetStates()`/`SetBlockStates()` etc).